roles/pulp_common/tasks/install_pip.yml

```diff
@@ -274,7 +274,7 @@
         django_location=$({{ pulp_install_dir }}/bin/pip3 show Django | grep Location | cut -d' ' -f2)
         sed -i 's/hashlib.md5()/hashlib.md5(usedforsecurity=False)/' "$django_location/django/db/backends/utils.py"
       when:
-        - "'pulp_devel' in role_names or 'pulp.pulp_installer.pulp_devel' in role_names"
+        - developer_user is defined and developer_user_home is defined
```

Comment:
As I'm dynamically including roles, I had to find another way to test if we are running for devel

Reply:
Hmm, how about "or" instead?